after-effects

```diff
@@ -650,9 +650,9 @@ function _fix_ubuntu_derivatives()
                                             code_name="xenial"
                                             _log_success "${_elementary} Changed codename to $code_name"
                                             ;;
-      juno )                                _log_and_show "${_elementary} This release of Elementary OS is based on Ubuntu 18.04 Bionic."
+      juno | hera)                          _log_and_show "${_elementary} This release of Elementary OS is based on Ubuntu 18.04 Bionic."
```

Comment:
Is the release stable? If not a warning should be added

Reply:
I have updated the table at `docs/faq/distros.md`.
The stable release of 5.1 "Hera"  was released on 3 December 2019.